sami2py/_core.py

```diff
@@ -282,8 +286,19 @@ def _generate_drift_info(fejer, ExB_drifts=None):
     """
     drift_info = _generate_fortran_bool(fejer)
     if not fejer:
-        if ExB_drifts.shape != (10, 2):
-            raise Exception('Invalid ExB drift shape!  Must be 10x2 ndarray.')
+        if ExB_drifts is None:
+            ExB_drifts = np.zeros((10, 2))
+
+        if isinstance(ExB_drifts, str) and ExB_drifts == 'default':
+            ExB_drifts = np.array([[-30, 0], [0, 0], [0, 0], [0, 0], [0, 0],
+                                  [0, 0], [0, 0], [0, 0], [0, 0], [0, 0]])
```

Comment:
Suggested for improved maintainability (ie, if we decide to change the number of fourier coefficients down the line)
```suggestion
            ExB_drifts = np.zeros((10, 2))
            ExB_drifts[0, 0] = -30
```